unpackaged/config/trailhead/profiles/Office Staff.profile

```diff
@@ -0,0 +1,4217 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Profile xmlns="http://soap.sforce.com/2006/04/metadata">
+    <applicationVisibilities>
+        <application>%%%NAMESPACE%%%Nonprofit_CRM</application>
+        <default>true</default>
+        <visible>true</visible>
+    </applicationVisibilities>
+    <classAccesses>
```

Comment:
There are a few new classes that should get added to all the profiles for access. These 3 have auraEnabled methods and definitely require explicit access for users.

- RD2_EnablementDelegate_CTRL
- RD2_VisualizeScheduleController
- UTIL_BatchJobProgress_CTRL